### PR TITLE
update quarkus version 2.9.1.Final -> 3.4.3

### PR DESCRIPTION
**Pull request by RasDeaks**

### CODE REVIEW

**Comment by RasDeaks** [comment]
About Exercice 11, since Multi are not supported anymore, should I update the readme which mentions this part.
See https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0#support-of-multi

File: code/exercise_011_Going_Reactive/src/main/java/com/lunatech/training/quarkus/ProductsResource.java [hunk 1]
```diff
@@ -15,7 +19,13 @@ public class ProductsResource {
...
     @GET
     public Multi<Product> products() {
-        return Product.streamAll();
+        return getAllProducts().toMulti()
+                .flatMap(list -> Multi.createFrom().iterable(list));
+    }
+
+    @WithTransaction
+    public Uni<List<Product>> getAllProducts(){
+        return Product.listAll();
```
Comment (wjglerum) [inline code/exercise_011_Going_Reactive/src/main/java/com/lunatech/training/quarkus/ProductsResource.java:28]: We just need the following right (like you have in subsequent exercises)?

```java
    @GET
    @WithTransaction
    public Uni<List<Product>> products(){
        return Product.listAll();
    }
```

**Review by wjglerum — COMMENTED** [review]
Looks like a pretty straightforward rewrite to Quarkus 3, thanks!

File: code/exercise_011_Going_Reactive/src/main/java/com/lunatech/training/quarkus/ProductsResource.java (hunk 1 above)
Comment (RasDeaks) [reply code/exercise_011_Going_Reactive/src/main/java/com/lunatech/training/quarkus/ProductsResource.java:28]: Good question.
In subsequent exercise I simplified that part by directly changing the signature from `Uni` to `Multi`.
Here, I decided to keep this return type as it was mentionned in the README of this exercice.
The thing is, I wasn't able to Annotate a method returning a `Multi`. I think we've got 2 side effect here (session not available in CDI anymore + panache `streamAll` method deletion) and that's why I separated this method in 2.
I'm not shure this is relevant. I think it's the same.
WDYT ? Should I delete the part where Multi are mentionned and aligne with subsequent exercise ?

About hibernate  Session : https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0#sessions-and-transactions
About Panache streamAll method : https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0#support-of-multi

Comment (wjglerum) [reply code/exercise_011_Going_Reactive/src/main/java/com/lunatech/training/quarkus/ProductsResource.java:28]: I would just simplify it here and use the same approach as you did in subsequent exercises and adjust the readme accordingly of needed.

Comment (RasDeaks) [reply code/exercise_011_Going_Reactive/src/main/java/com/lunatech/training/quarkus/ProductsResource.java]: Done

**Review by wjglerum — APPROVED** [review]
nice work!

**Pull request merged**